src/getTasks.js

```diff
@@ -9,61 +10,88 @@ function createMyElement(para) {
   return document.createElement(para);
 }
 
-export function addtoLocal() {
-  tasks.forEach((elem) => {
-    const key = elem.index;
-    localStorage.setItem(key, JSON.stringify(elem));
-  });
-}
+const enterTaskInput = getMyElement('#enter-tasks');
+const taskForm = getMyElement('#add-task-form');
 
-function printTasks() {
-  addtoLocal();
-  Object.keys(localStorage).forEach((key) => {
-    if (key) {
-      const data = JSON.parse(localStorage.getItem(key));
-      if (data) {
-        const li = createMyElement('li');
-        li.innerHTML = data.description;
-        const ul = getMyElement('#inner-tasks');
+function printTasks(e) {
+  e.preventDefault();
+  const li = createMyElement('li');
+  const spanDec = createMyElement('span');
+  spanDec.innerHTML = enterTaskInput.value;
+  spanDec.className = 'span-desc';
+  li.appendChild(spanDec);
+  const ul = getMyElement('#inner-tasks');
 
-        const checkBox = createMyElement('input');
-        checkBox.type = 'checkbox';
-        checkBox.name = 'checkbox';
-        checkBox.className = 'check-box';
-        li.prepend(checkBox);
+  const checkBox = createMyElement('input');
+  checkBox.type = 'checkbox';
+  checkBox.name = 'checkbox';
+  checkBox.className = 'check-box';
+  li.prepend(checkBox);
 
-        const dots = createMyElement('span');
-        dots.innerHTML = '⋮';
-        dots.className = 'dot-line';
-        li.appendChild(dots);
+  const dots = createMyElement('span');
+  dots.innerHTML = '⋮';
+  dots.className = 'dot-line';
+  li.appendChild(dots);
+
+  dots.addEventListener('click', () => {
+    if (tasks.length > 0) {
+      tasks.map((task) => {
+        if (task.description === spanDec.innerHTML) {
+          if (dots.innerHTML === '⋮') {
+            dots.innerHTML = '&#x1F4BE;';
+            spanDec.contentEditable = 'true';
+            spanDec.addEventListener('input', () => {
+              task.description = spanDec.innerHTML;
```

Comment:
- [ ] I think it is better if you make the description an `innerText` rather than an `innerHTML` because when it is updated actions like hitting enter would be taken as characters. When I update the description and hit enter it adds the character `<br>` to the updated description in the local storage. Kindly make changes here! :pray: 

![Screenshot from 2021-10-30 00-13-28](https://user-images.githubusercontent.com/64894927/139502862-d78d6cbb-b2ca-4a22-b97e-f68f9a06af01.png)

Reply:
This issue is well addressed. Nice job! :pray: